server/plugin.go

```diff
@@ -426,6 +428,38 @@ func (p *Plugin) OnActivate() error {
 		}
 	}
 
+	remoteID, err := p.API.RegisterPluginForSharedChannels(model.RegisterPluginOpts{
+		Displayname:  "ms_teams_connect",
```

Comment:
Propose matching the pending name:
```suggestion
		Displayname: "MS Teams Plugin",
```